shared/common-adapters/plain-input.desktop.tsx

```diff
@@ -159,6 +159,13 @@ class PlainInput extends React.PureComponent<InternalProps> {
 
   _onFocus = () => {
     this.props.onFocus && this.props.onFocus()
+    this.props.selectTextOnFocus &&
+      setTimeout(() =>
```

Comment:
why? also this is likely not safe on unmount

Reply:
I think, it didn't work within the same tick. Had to defer  it like this. I guess we could check mounted state here?